Documentation/ApiOverview/FlexForms/Index.rst

```diff
@@ -60,116 +66,80 @@ Steps to perform (extension developer)
-         // ctype
-         'accordion'
-      );
-
-   Finally, according to "Configuration of the displayed order of fields in FormEngine
-   and their tab alignment." the field containing the FlexForm still needs to be
-   added to the `showitem` directive.
-   The following example shows line from the accordion element of the Bootstrap Package.
-
-   .. code-block:: php
-      :caption: Add in file EXT:your_extension/Configuration/TCA/Overrides/tt_content.php
-      :emphasize-lines: 11
-
-      // Configure element type
-      $GLOBALS['TCA']['tt_content']['types']['accordion'] = array_replace_recursive(
-         $GLOBALS['TCA']['tt_content']['types']['accordion'],
-         [
-            'showitem' => '
-               --div--;General,
-               --palette--;General;general,
-               --palette--;Headers;headers,
-               tx_bootstrappackage_accordion_item,
-               --div--;Options,
-               pi_flexform'
-         ]
-      );
+#.  Create configuration schema in :ref:`T3DataStructure <t3ds>` format (XML)
+
+    **Example:**
+
+    ..  include:: /CodeSnippets/FlexForms/Examples/PluginHaikuList.rst.txt
+
+
+#.  The configuration schema is attached to one or more plugins in the folder
+    :file:`Configuration/TCA/Overrides` of an extension.
+
+    Example for the FlexForm registration of a basic plugin:
+
+    ..  include:: /CodeSnippets/FlexForms/Examples/PluginHaikuListRegistration.rst.txt
+
+    ..  versionadded:: 12.0
+        The method :php:`ExtensionUtility::registerPlugin()` returns the plugin signature.
+
+    In when registering Extbase plugins you can use the return value of
+    :php:`ExtensionUtility::registerPlugin()` to figure out the plugin
+    signiture to use:
+
+    ..  code-block::
+        :caption: EXT:blog_example/Configuration/TCA/Overrides/tt_content.php (Excerpt)
+
+        $pluginSignature = ExtensionUtility::registerPlugin(
+            'blog_example',
+            'Pi1',
+            'A Blog Example',
+        );
+        $GLOBALS['TCA']['tt_content']['types']['list']['subtypes_addlist'][$pluginSignature]
+            = 'pi_flexform';
+        ExtensionManagementUtility::addPiFlexFormValue(
+            $pluginSignature,
+            'FILE:EXT:blog_example/Configuration/FlexForms/PluginSettings.xml'
+        );
+
+
+    If you are using a content element instead of a plugin, the example
+    looks like this:
+
+    ..  code-block:: php
+        :caption: EXT:my_extension/Configuration/TCA/Overrides/tt_content.php
+
+        \TYPO3\CMS\Core\Utility\ExtensionManagementUtility::addPiFlexFormValue(
+            // 'list_type' does not apply here
+            '*',
+            // FlexForm configuration schema file
+            'FILE:EXT:example/Configuration/FlexForms/Registration.xml',
+            // ctype
+            'accordion'
+        );
+
+    Finally, according to "Configuration of the displayed order of fields in FormEngine
```

Comment:
Where can I find this "according to" section? Maybe links to it.

Reply:
The text has been there before, all I did was change indentation. This is out of scope and can be done in a follow up...